outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "metallb" {
   value = module.metallb
-}
\ No newline at end of file
+}
+
+output "cloud_provider_equinix_metal" {
+  value = module.cloud_provider_equinix_metal
+}
+
+output "cloud_provider_equinix_metal_prerequisites" {
+  value = try(module.cloud_provider_equinix_metal.0.prerequisites, [])
```

Comment:
In order for prerequisites to work, they have to be exposed as a separate output in the wrapper module.  Trying to reference `module.addons.cloud_provider_equinix_metal.prerequisites` leads to a cycle error.

I'm leaning towards breaking prerequisites into a separate PR so we can continue to try out the concept, and just documenting the one CPEM prerequisite in the README for that module for now.

Reply:
I ended up deciding to keep this in the add-on for now.  Since this project is experimental, this feature can easily be removed if we find that it's not worth keeping in the future.